packages/next-codemod/transforms/__testfixtures__/next-request-geo-ip/destructure-alias.output.ts

```diff
@@ -0,0 +1,13 @@
+// @ts-nocheck
+import { type NextRequest, NextResponse } from "next/server";
+
+import { geolocation, ipAddress } from "@vercel/functions";
```

Comment:
The testing output files have some `import` with a line break, and this is [could-not-fix behavior](https://github.com/benjamn/recast/issues/371#issuecomment-368260356) of `recast` (dependency of `jscodeshift`). This occurs when there's a top-level comment and the target to `insertAfter` is right below it.

```ts
// @ts-nocheck <-- comment here
import { type NextRequest, NextResponse } from "next/server";

import { geolocation, ipAddress } from "@vercel/functions"; // insertAfter
```